modules/va_facilities/lib/va_facilities/csv_serializer.rb

```diff
@@ -57,7 +57,11 @@ def self.phone_attrs(object)
 
     def self.hours_attrs(object)
       hours = object.hours
-      %w[monday tuesday wednesday thursday friday saturday sunday].map { |day| hours[day] }
+      if object.facility_type_prefix == 'vc'
+        %w[monday tuesday wednesday thursday friday saturday sunday].map { |day| hours[day] }
```

Comment:
Would it be simpler if we modified VetCenter days as they're being read in from ArcGIS so that we no longer need this type of condition in our system? I bet there are other instances of exceptions for different day names around, too.

Reply:
I would like this, but I haven't looked into the haul necessary to make that change.

Reply:
I will make a tech debt ticket. I think it shouldn't be too bad of a change